app/components/container/container.html

```diff
@@ -300,6 +300,22 @@
         <div class="pagination-controls">
           <dir-pagination-controls></dir-pagination-controls>
         </div>
+        <hr />
+        <form class="form-horizontal">
+          <!-- network-input -->
+          <div class="row">
+            <label for="container_network" class="col-sm-4 col-lg-2 control-label text-left">Join a Network</label>
```

Comment:
Can you review the `col-sm` on each attribute here? The total should be 12. See the actual result on a 768*1024 device.

![portainer 1](https://user-images.githubusercontent.com/5485061/27622377-003ab3e6-5bd7-11e7-8c48-ebbbca6a9093.png)